src/components/TermandPrivacy/InformativeClausule.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { RegulationsCards } from './RegulationsCards';
+
+export const InformativeClause = () => {
```

Comment:
Why didn't You use an simplefied version? () => (<div/>) ?
And where is FC type for this functional component?